integration-test/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -32,8 +31,9 @@
   @Test
   public void cachemanagerExample() {
     // tag::java7Example[]
-    try(CacheManager cacheManager = newCacheManagerBuilder() // <1>
-      .withCache("preConfigured", newCacheConfigurationBuilder(Long.class, String.class, heap(10))) // <2>
+    try(CacheManager cacheManager = CacheManagerBuilder.newCacheManagerBuilder() // <1>
```

Comment:
That's not right. The goal was to show that we can use static imports. So remove it doesn't work